osquery/logger/logger.cpp

```diff
@@ -206,9 +200,6 @@ class BufferedLogSink : public google::LogSink, private boost::noncopyable {
    */
   std::atomic<bool> enabled_{false};
 
-  /// Boolean to help the logger disabler, no need to take action if not active.
-  bool active_{false};
```

Comment:
There is no need for active_.
active_ was used only in setUp enable disable.

However, setup does not need active_. setUp is called only once. not even testing mechanisms need to touch it.

As setUp does not need it, in the case for the enable/disable this variable is redundant.